packages/firebase_auth/ios/firebase_auth.podspec

```diff
@@ -16,7 +16,7 @@ Firebase Auth plugin for Flutter.
   s.public_header_files = 'Classes/**/*.h'
   s.ios.deployment_target = '8.0'
   s.dependency 'Flutter'
-  s.dependency 'Firebase/Auth'
+  s.dependency 'Firebase/Auth', '~> 5'
```

Comment:
This is meaningless. To lock to 5.x, use '~> 5.0'.  See https://guides.cocoapods.org/syntax/podspec.html#dependency